btf/handle.go

```diff
@@ -119,3 +119,33 @@ func (it *HandleIterator) Next(handle **Handle) bool {
 func (it *HandleIterator) Err() error {
 	return it.err
 }
+
+// FindHandle returns the first handle for which predicate returns true.
+//
+// Requires CAP_SYS_ADMIN.
+//
+// Returns ErrNotFound if predicate never returns true or if there is no BTF
```

Comment:
Nit: You return a wrapped version of `ErrNotFound`, this might give users the impression they can use `err != ErrNotFound` to check the output while they need to check with `error.Is()`. So I would reword the comment, remove it or change the return statement. Which ever suites best

Reply:
Good point, I'll amend the doc.